### DISCUSSION TITLE
GameCube HW2 - 8bitdo Pro 2

### DISCUSSION THREAD

**Discussion by haydnkilyan** [darthcloud/BlueRetro#899] [Q&A]
Hello, always used a Pro 2 with my GC internal mode. Used the 8bitdo SN30 Src label and it worked fine (there is no specific Pro 2 option). Latest firmware has made the button mapping all incorrect - start is now mapped to Z for example- not sure why this change has occurred

**Comment by haydnkilyan** [comment]
Rolled back to 1.8.3 firmware and it works fine

**Comment by darthcloud** [comment]
Can you take me a screenshot of the controller name when paired to windows. In the Bluetooth manager.

Screenshot is best since I need the correct case of every character.

**Comment by darthcloud** [comment]
I just realized I already fixed this for next version.

See beta build posted in this thread:
https://github.com/darthcloud/BlueRetro/issues/857

**Reply by haydnkilyan** [reply]
That is great, was going to look at this today, I'll try the beta firmware. The 1.9.1 works fine on my humble Bazooka PC Engine Adapter with the 8Bitdo controller ..... and I do have the Neo BT too (but haven't updated that for a while)

**Comment by darthcloud** [comment]
[Fixed in v1.9.2](https://github.com/darthcloud/BlueRetro/releases/tag/v1.9.2)
